components/contribution-flow/StepDetailsCrypto.js

```diff
@@ -123,12 +127,12 @@ const StepDetailsCrypto = ({ onChange, data, collective }) => {
         inputMode="decimal"
         defaultValue={amount}
         onChange={({ target }) => {
-          const amount = parseFloat(target.value);
+          const amount = parseFloat(target.value).toFixed(2);
```

Comment:
I'm surprised by this, do we really want to limit to 2 decimals? Let's take BTC, 0.01BTC ~= $150 therefore wouldn't we prevent contributions under $150? Looking at the issue I don't really get what it's trying to solve

Reply:
Yes, this seems to be a mistake, thanks for pointing it out. I've corrected this.